source/index.html.erb

```diff
@@ -88,24 +88,47 @@ title: Donate to Ada Developers Academy
 </div>
 <div class="gap" id="loan-fund">
   <div class="container about">
-    <h1>Ada's Loan Fund</h1>
-    <p>Morbi a neque enim. Vivamus viverra turpis ac turpis convallis, in congue ipsum mollis. Sed pretium ornare lorem sed eleifend. Cras varius nec arcu ac porttitor. Mauris venenatis facilisis turpis, non egestas sem suscipit nec.</p>
-    <div class="about--container">
-      <img src="images/icons-01.png">
-      <p>Etiam vulputate ex et mauris tincidunt posuere. Maecenas suscipit fermentum odio lacinia lacinia. Maecenas aliquet purus vestibulum justo venenatis rhoncus. Maecenas lorem dui, tristique vel elit sit amet, consequat auctor lorem. Aenean finibus velit nisl.</p>
+    <h1>Ada Developers Academy: The game changer</h1>
+    <p>This program is a game changer for our students and the tech industry at large.</p>
+    <div>
+      <h3>Industry Supported</h3>
+      <ul>
+        <li>60+ sponsor companies</li>
+        <li>74% sponsor return rate within one year of initial sponsorship</li>
+        <li>29% of current sponsors are sponsoring more than one person from the same cohort</li>
+        <li>98% candidate placement rate</li>
+      </ul>
     </div>
-    <div class="about--container">
-      <img src="images/icons-02.png">
-      <p>Vivamus pulvinar ipsum in tempor molestie. Phasellus eu neque neque. Mauris vulputate, metus eget euismod ornare, nunc nisl tempus diam, sed eleifend ligula ipsum ut erat.</p>
+    <div>
+      <h3>Student Diversity Matters</h3>
+      <ul>
+        <li>Age range typically 21 - 49</li>
+        <li>Range of previous careers – flight attendant, barista, teacher, social worker, scientist, researcher, retail</li>
+        <li>Ethnicity - African American, Asian Latinx, Caucasian, Native American</li>
+        <li>Backgrounds – Low income/homeless, LGBTQ</li>
```

Comment:
I asked Nicole for specific updates to be made to this copy that I don't see - so let's hang tight on this until I follow-up on that